airflow/hooks/drill_hook.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from pydrill.client import PyDrill
+from pydrill.exceptions import PyDrillException
+from airflow.hooks.dbapi_hook import DbApiHook
+
+
+class DrillHook(DbApiHook):
+    """
+    Interact with Drill through  PyDrill!
+
```

Comment:
Such a docstring may not be info-rich enough and seems typo inside.

Reply:
I will improve it but I did write it by comparing with `presto_hook`